kod/util/rntmaint.kod

```diff
@@ -263,7 +265,7 @@ messages:
       % Is the item a key copy?  If so, destroy it.
       for oItem in itemList
       {
-         if GetClass(oItem) = &RoomKeyCopy
+         if IsClass(oItem, &RoomKeyCopy) AND Send(oItem, @IsKeyCopy)
```

Comment:
This IsKeyCopy looks pretty suspicious.  Is it not redundant with the IsClass check?

Reply:
Yeah it's a bit weird at a glance, but due to the way that the key classes were originally set up, RoomKeyCopy is the superclass of RoomKey and now RoomKeySecureCopy, so it'd be true for all 3 key types when it should be false for RoomKey.  So basically, that check is asking "are you a room key of any kind?  Okay great, now I can ask if you're a room key copy without causing errors."